pkg/microservice/systemconfig/core/features/repository/mongodb/feature.go

```diff
@@ -0,0 +1,52 @@
+package mongodb
+
+import (
+	"context"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+
+	"github.com/koderover/zadig/pkg/microservice/systemconfig/config"
+	"github.com/koderover/zadig/pkg/microservice/systemconfig/core/features/repository/models"
+	mongotool "github.com/koderover/zadig/pkg/tool/mongo"
+)
+
+type FeatureColl struct {
+	*mongo.Collection
+
+	coll string
+}
+
+func NewFeatureColl() *FeatureColl {
+	name := models.Feature{}.TableName()
+	coll := &FeatureColl{Collection: mongotool.Database(config.MongoDatabase()).Collection(name), coll: name}
+
+	return coll
+}
+
+func (c *FeatureColl) GetCollectionName() string {
+	return c.coll
+}
+func (c *FeatureColl) EnsureIndex(ctx context.Context) error {
+	return nil
+}
+
+func (c *FeatureColl) UpdateOrCreateFeature(obj *models.Feature) error {
```

Comment:
If statements in the method need `context`, it's better to add a `ctx context.Context `argument to the method.

Reply:
our project haven't use context to control  life circle ,trace